usl_pipeline/usl_lib/usl_lib/shared/wps_data.py

```diff
@@ -311,5 +322,32 @@ class Unit(Enum):
                 "type": ScalingType.NONE,
             },
         },
+        # [Derived] Solar Time from UTC (Cyclic feature) in Minutes of Day (MIN).
+        # Sine Component of Solar Time
+        "SOLAR_TIME_SIN": {
+            "unit": Unit.NONE,
+            "scaling": {
+                "type": ScalingType.GLOBAL,
+                "min": -1,
+                "max": 1,
+            },
+        },
+        # [Derived] Solar Time from UTC (Cyclic feature) in Minutes of Day (MIN).
+        # Cosine Component of Solar Time
+        "SOLAR_TIME_COS": {
+            "unit": Unit.NONE,
+            "scaling": {
+                "type": ScalingType.GLOBAL,
+                "min": -1,
+                "max": 1,
+            },
+        },
+        # LU_INDEX
```

Comment:
Lets move this one next to LANDUSEF (we can do this now bc we haven't generated any labels yet so lets organize it how we like it before it's solidified)

Reply:
Done! :)